metadata-ingestion/src/datahub/ingestion/transformer/add_dataset_ownership.py

```diff
@@ -102,7 +98,7 @@ def transform_aspect(
 
 
 class DatasetOwnershipBaseConfig(TransformerSemanticsConfigModel):
-    ownership_type: Optional[str] = OwnershipTypeClass.DATAOWNER
+    ownership_type: Optional[str]
```

Comment:
we don't need to do this in this PR, but it might simplify some of the `cast` calls away

```suggestion
    ownership_type: str = OwnershipTypeClass.DATAOWNER
```

Reply:
Reverted to that (changed it in first place due to addition of another field, now removed).